mmseg/models/decode_heads/decode_head.py

```diff
@@ -33,8 +33,11 @@ class BaseDecodeHead(BaseModule, metaclass=ABCMeta):
                 a list and passed into decode head.
             None: Only one select feature map is allowed.
             Default: None.
-        loss_decode (dict): Config of decode loss.
-            Default: dict(type='CrossEntropyLoss').
+        loss_decode (dict | Sequence[dict]): Config of decode loss.
+             e.g. dict(type='CrossEntropyLoss'),
+             [dict(type='LossName_1', name='loss_1'),
+              dict(type='LossName_2', name='loss_2')]
+            Default: dict(type='CrossEntropyLoss', name='ce').
```

Comment:
```suggestion
            Default: dict(type='CrossEntropyLoss').
```
The default in `__init__`.